x-pack/plugins/uptime/public/components/monitor/synthetics/executed_step.tsx

```diff
@@ -12,80 +12,104 @@ import { CodeBlockAccordion } from './code_block_accordion';
 import { StepScreenshotDisplay } from './step_screenshot_display';
 import { StatusBadge } from './status_badge';
 import { Ping } from '../../../../common/runtime_types';
+import { StepDetailLink } from '../../common/step_detail_link';
 
 const CODE_BLOCK_OVERFLOW_HEIGHT = 360;
 
 interface ExecutedStepProps {
   step: Ping;
   index: number;
+  checkGroup: string;
 }
 
-export const ExecutedStep: FC<ExecutedStepProps> = ({ step, index }) => (
-  <>
-    <div style={{ padding: '8px' }}>
-      <div>
-        <EuiText>
-          <strong>
-            <FormattedMessage
-              id="xpack.uptime.synthetics.executedStep.stepName"
-              defaultMessage="{stepNumber}. {stepName}"
-              values={{
-                stepNumber: index + 1,
-                stepName: step.synthetics?.step?.name,
-              }}
-            />
-          </strong>
-        </EuiText>
+export const ExecutedStep: FC<ExecutedStepProps> = ({ step, index, checkGroup }) => {
+  return (
+    <>
+      <div style={{ padding: '8px' }}>
```

Comment:
opportunity to use euiStyled :)

Reply:
Could be yep. This inline style was already on `master`: https://github.com/elastic/kibana/blob/master/x-pack/plugins/uptime/public/components/monitor/synthetics/executed_step.tsx#L25, I just kept it as is incase there was a reason.

Reply:
++